Airspeed: move from development to common

IamPete1 opened the pull request:
This message is supported by ArduPilot 4.6 stable release, so we should move it out of development.xml. 

This is just a copy and paste, there is no change to the message or the enum.

hamishwillee commented on message_definitions/v1.0/common.xml:
```diff
@@ -7409,7 +7418,14 @@
+      <field type="uint8_t" name="id" instance="true">Sensor ID.</field>
+      <field type="float" name="airspeed" units="m/s">Calibrated airspeed (CAS).</field>
+      <field type="int16_t" name="temperature" units="cdegC">Temperature. INT16_MAX for value unknown/not supplied.</field>
+      <field type="float" name="raw_press" units="hPa">Raw differential pressure. NaN for value unknown/not supplied.</field>
```
Let's use metadata for these
```suggestion
      <field type="int16_t" name="temperature" units="cdegC" invalid="INT16_MAX">Temperature.</field>
      <field type="float" name="raw_press" units="hPa" invalid="NaN">Raw differential pressure.</field>
```

hamishwillee reviewed (commented):
Thanks @IamPete1 
- Did you encounter any wrinkles or changes that needed to be made when implementing?
- Is this just used for display?
- This is kind of last chance for modification. Was there anything else you would have done with this following testing?

@peterbarker @julianoes Any reason not to merge?

peterbarker commented:
On Tue, 21 Oct 2025, Hamish Willee wrote:

> @peterbarker @julianoes Any reason not to merge?

If we another implementation it could go into standard.xml?

julianoes commented:
> This message is supported by ArduPilot 4.6 stable release, so we should move it out of development.xml.

FWIW, that's not generally the process. Something should be moved out of development.xml before a release happens. There shouldn't be pressure to merge something "because it has been released".

I strongly suggest that ArduPilot does not ship with development.xml in releases because from what I understand ArduPilot does not tend to break things for users!

In this case I think it's fine.

peterbarker reviewed (approved):
This message does work well.

IamPete1 commented:
> * Did you encounter any wrinkles or changes that needed to be made when implementing?
> * Is this just used for display?
> * This is kind of last chance for modification. Was there anything else you would have done with this following testing?

This is already a second iteration, we updated it in https://github.com/mavlink/mavlink/pull/1900. (I did mess up the enum, but we fixed that in https://github.com/mavlink/mavlink/pull/2211)

Yeah, its just for display, there is no way to send airspeed message from anything but the first sensor without this. AP has been using a named float for airspeed two for a long time. We now support more sensors (upto 6) so we needed something better.



> FWIW, that's not generally the process. Something should be moved out of development.xml before a release happens. There shouldn't be pressure to merge something "because it has been released".

I agree, but I was overcome by events, only just had time to get back to it.
